jis-plugins/src/main/java/pl/edu/uj/JImageStream/filters/ConvolutionFilter.java

```diff
@@ -28,8 +28,8 @@ public void apply(int x, int y) {
                 outputGreen += (int) ((neighbours.get(i).getGreen()) * kernel[i]);
                 outputBlue += (int) ((neighbours.get(i).getBlue()) * kernel[i]);
             }
-            setPixel(x, y, new Pixel(outputRed, outputGreen, outputBlue, 255));
-
+            int alpha = getPixel(x, y).getAlpha();
```

Comment:
There is - when alpha channel is changed with the same algorithm as other channels, RobertsCross filter's produce non-visible content.